src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarShippingInformation.tsx

```diff
@@ -122,6 +121,8 @@ export const ArtworkSidebarShippingInformationFragmentContainer =
         isUnlisted
         priceIncludesTaxDisplay
         shippingOrigin
+        artsyShippingDomestic
```

Comment:
Maybe worth cleaning this up as those are not needed in this component now that we've moved the check into the parent component.